strongbox-aql/src/test/java/org/carlspring/strongbox/aql/AqlParserTest.java

```diff
@@ -227,4 +227,7 @@ public void testInvalidQuery()
         assertThat(Pair.with(1, 78).equals(errorPositionList.get(3))).isTrue();
     }
 
+    private void logQueryAndAqlParser(String query, AqlQueryParser aqlParser) {
```

Comment:
1) We don't need to have a new method for this. We should still be using the respective `logger` methods directly.
2) Could you please follow our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)